src/hotspot/share/gc/z/zHeuristics.cpp

```diff
@@ -63,7 +63,7 @@ bool ZHeuristics::use_per_cpu_shared_small_pages() {
   // Use per-CPU shared small pages only if these pages occupy at most 3.125%
   // of the max heap size. Otherwise fall back to using a single shared small
   // page. This is useful when using small heaps on large machines.
-  const size_t per_cpu_share = significant_heap_overhead() / ZCPU::count();
+  const size_t per_cpu_share = (MaxHeapSize * 0.03125) / ZCPU::count();
```

Comment:
The proposed change just reverts what appears to have been an intentional code
change in JDK-8307058. That change should have updated the comment.  So I
don't think this change should be made.  The comment should be fixed instead.

Reply:
Agreed.

Reply:
Should I change the title of this issue and fix the comment in this patch? Or open another issue and PR to fix it?

Reply:
I update the code which only changes the comment, and revise the title.